docs/_sidebar.md

```diff
@@ -50,9 +50,11 @@
     - [vela workloads](/en/cli/vela_workloads.md)
   - Traits
     - [vela traits](/en/cli/vela_traits.md)
-    - [vela manualscale](/en/cli/vela_manualscaler.md)
-    - [vela scale](/en/cli/vela_scale.md)
+    - [vela scaler](/en/cli/vela_scaler.md)
```

Comment:
`vela scale` is better. I changed it to a verb.

Reply:
sure， you could raise a PR for that